commit 27b2f1304b352cc15901b0b5dff90be469176e9c
Author: Trustin Lee <trustin@gmail.com>
Date:   Mon Mar 11 09:11:11 2013 +0900

    Why don't CI services upgrade to Maven 3.0.5?
    
    Do they just wait for someone to exploit the vulnerability?

diff --git a/pom.xml b/pom.xml
index 285f02f..3f594eb 100644
--- a/pom.xml
+++ b/pom.xml
@@ -273,7 +273,7 @@
                   <version>[1.7.0,)</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>[3.0.5,)</version>
+                  <version>[3.0.4,)</version>
                 </requireMavenVersion>
               </rules>
             </configuration>
